source/sync/migrate-sync-modes.txt

```diff
@@ -285,6 +279,31 @@ start over from the beginning with future migration attempts.
    This will revert your migration. You will need to migrate again if you 
    want to enable Flexible Sync in the future.
 
+.. _realm-sync-migrate-after:
+
+After Backend Migration
+-----------------------
+
+Your migrated backend uses a partition key field in your Realm object models
+to map the Partition-Based Sync client objects to Flexible Sync equivalents on
+the backend. If your object models don't have a partition key field, the
+backend automatically injects one into every table the Partition-Based Sync
+client creates.
+
+The injected field follows this structure: ``<yourPartitionKey> == 
+<yourPartitionValue>``. Both the field key and value are derived from your 
+Partition-Based Sync client configuration.
+
+There are some other characteristics of the injected field you should be aware of:
+
+- The injected field does not sync back to the client that created the table.
+  This means the originating client won't be able to read the injected
+  ``partitionKey`` field.
+- The Partition-Based Sync client automatically creates sync subscriptions for 
```

Comment:
this is not really a characteristic of the injected field, so it should probably be pulled out of this list

Reply:
Ah, yes. Thank you for noticing this discrepancy. I've re-organized a bit.